README.mkdn

```diff
@@ -25,13 +25,13 @@ Planned features:
 [Discourse]: http://www.discourse.org/
 [Forem]: https://www.github.com/radar/forem
 
-Thredded works with SQLite, MySQL (v5.6.4+), and PostgreSQL.
-Thredded has no infrastructure dependencies other than the database and, if configured in the parent application,
-the ActiveJob backend dependency such as Redis.
-Currently only MRI Ruby 2.2+ is supported. We would love to support JRuby and Rubinius as well.
+Thredded works with SQLite, MySQL (v5.6.4+), and PostgreSQL. Thredded has no infrastructure
```

Comment:
You make a good point. If that's the usual norm I need to update my vimrc to adhere to this.